app/controllers/webhooks/base_controller.rb

```diff
@@ -0,0 +1,16 @@
+class Webhooks::BaseController < ApplicationController
+  rescue_from ApiErrors::StandardError, with: :stripe_error
```

Comment:
Another thing about these `rescue_from`declarations. We would need to be rescuing that particular error. This is an error class that I created to standardize how we're reporting errors from our API to the consuming client. From what I've seen in this PR, we're not going to be throwing these errors.

Really, we should be rescuing errors we know that are going to be thrown, like those on [line 3](https://github.com/project-signal/project-signal/blob/aafe2dc542f6600330bb8f87e4c7bd35f1edc723/app/controllers/webhooks/base_controller.rb#L3). However, we should be a little more selective about which errors respond with a 4XX or 5XX. For example, if the event passed to us has a malformed event id that does not exist on Stripe, Stripe will throw a `Stripe::InvalidRequestError` with an http status of 404. This is not an error we want to respond back with a 4XX. 

They say that any failed event will be retried once an hour for 3 days.

![image](https://cloud.githubusercontent.com/assets/15848772/19838399/de7e8d08-9ea4-11e6-992d-ad410c1cc5ae.png)

This is good because if our db or server is down it'll keep retrying. But for those we events we know we don't want processed, we should respond with 2XXs as well.

Another note is that these `rescue_from`s are pretty specific to Stripe webhooks. We should move them outside of here. I'd create a base Stripe webhook controller to specify these `rescue_from`s.